src/js/controllers/imageCtrl-test.js

```diff
@@ -0,0 +1,125 @@
+describe('imagesCtrl', function() {
+    var $rootScope, $controller, imagesCtrl, images, vm, $q;
+
+    beforeEach(module('flickrPOC'));
+
+    beforeEach(module(function($provide){
+        $provide.service('images', function($q){
+            this.loadPics = sinon.stub().returns($q.resolve({message: 'loadPics Working fine'}));
+        });
+
+    }));
+
+    beforeEach(inject(function (_$rootScope_, _$controller_, _images_, _$q_) {
+        $rootScope = _$rootScope_.$new();
+        $controller = _$controller_;
+        images = _images_;
+        $q = _$q_;
+
+        vm = $controller('imagesCtrl');
+
+    }));
+
+    describe('imagesCtrl', function(){
+        beforeEach(function () {
+            images.loadPics.returns($q.resolve(
+                {
+                    contentType: "text/json",
+                    status: "success",
+                    photos: {
+                        photo: [
```

Comment:
I'm not familiar with the Flickr API responses, but this looks strange. **Photos** is an object, and **photo** is an array. Seems to me that the array should have the plural label. Can you confirm this response object is correct?